pkg/cmd/server/start.go

```diff
@@ -301,11 +326,17 @@ func start(cfg *config, args []string) error {
 			osmaster.DeployerOSClientConfig = osClientConfig
 		}
 
+		// TODO: make anonymous auth optional
+		// TODO: make AllAnonymous group configurable (and possibly scoped under a system-controlled group prefix?)
```

Comment:
I'm for a system-controlled group prefix to avoid collisions.  I'm against making it configurable.  If the group is in its own prefix, there is no risk of collisions and no real benefit to making it configurable.  Predictability makes it safer to write rules against it.